modules/viewing-stats-collected-kubernetes-nmtate-op.adoc

```diff
@@ -0,0 +1,102 @@
+// This is included in the following assemblies:
+//
+// networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-stats-collected-kubernetes-nmtate-op_{context}"]
+= Viewing metrics collected by the Kubernetes NMState Operator
+
+The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and expose them as ready-to-use metrics. As a use case for viewing metrics, consider a situation where you created a `NodeNetworkConfigurationPolicy` custom resource and you want to confirm that the policy is active.
+
+[NOTE]
+====
+The `kubernetes_nmstate_features_applied` metrics are not an API and might change between {product-title} versions.
+====
+
+In the *Developer* and *Administrator* perspectives, the Metrics UI includes some predefined CPU, memory, bandwidth, and network packet queries for the selected project. You can run custom Prometheus Query Language (PromQL) queries for CPU, memory, bandwidth, network packet and application metrics for the project.
+
+The following example demonstrates a `NodeNetworkConfigurationPolicy` manifest example that is applied to an {product-title} cluster:
+
+[source,yaml]
+----
+# ...
+interfaces:
+  - name: br1
+    type: linux-bridge
+    state: up
+    ipv4:
+      enabled: true
+      dhcp: true
+      dhcp-custom-hostname: foo
+    bridge:
+      options:
+        stp:
+          enabled: false
+      port: []
+# ...
+----
+
+The `NodeNetworkConfigurationPolicy` manifest exposes metrics and makes them available to the Cluster Monitoring Operator (CMO). The following example shows some exposed metrics:
+
+[source,terminal]
+----
+controller_runtime_reconcile_time_seconds_bucket{controller="nodenetworkconfigurationenactment",le="0.005"} 16
+controller_runtime_reconcile_time_seconds_bucket{controller="nodenetworkconfigurationenactment",le="0.01"} 16
+controller_runtime_reconcile_time_seconds_bucket{controller="nodenetworkconfigurationenactment",le="0.025"} 16
+...
+# HELP kubernetes_nmstate_features_applied Number of nmstate features applied labeled by its name
+# TYPE kubernetes_nmstate_features_applied gauge
+kubernetes_nmstate_features_applied{name="dhcpv4-custom-hostname"} 1
+----
```

Comment:
I'll skip the concept module based on the fact that I have very little information to add to it and it would cause duplication of information in the procedure and concept module.